python/lsst/ts/Dome/dome_csc.py

```diff
@@ -361,7 +362,45 @@ async def config(self, _data):
             The contents of this parameter will be defined soon.
         """
         self.assert_enabled()
```

Comment:
Do you need to check enabled for non-SAL commands?

Reply:
Good question and it is one that I have been struggling with. Currently the mock_controller only gets started if the DCS state is disabled or enable. There doesn't seem to be an "assert_disabled_or_enabled()" function in SALobj.

Reply:
Correct. There is an `enabled_or_disabled` property and it's easy enough to explicitly test for the desired states. (Either way, it's best to raise `salobj.ExpectedError` to avoid a pointless traceback.)

Reply:
I have removed this assert for now, as well in other methods that are not being called by SAL directly. It is clear to me that a discussion is needed about configuring this CSC and until then I will leave the method in place to not have wasted time.